base/collect_database_test.go

```diff
@@ -41,9 +47,7 @@ type collectDatabaseTemplateInput struct {
 	to run, the metrics files are copied to a temprorary directory and the current
 	working directory is changed to the temporary directory
 */
-func GoToTempDirectoryAndCopyMetricsFile(t *testing.T, test func()) error {
-	metricsFileName := testCe + experimentMetricsPathSuffix
-
+func GoToTempDirectoryAndCopyMetricsFile(t *testing.T, metricFileName string, test func()) error {
```

Comment:
Change `GoToTempDirectoryAndCopyMetricsFile()` so that it can use different metrics file for different tests